databases/config/database.js

```diff
@@ -14,7 +14,8 @@ module.exports = {
     define: {
       timestamps: true,
       freezeTableName: true
-    }
+    },
+    logging: false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change to set `logging: false` will suppress all Sequelize logging. This might not be ideal in a staging environment where you might want to debug issues related to database operations. Consider making this configurable via an environment variable so that it can be toggled on and off as needed.
```diff
-    logging: false
+    logging: process.env.DB_LOGGING === 'true'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->